.github/workflows/main.yml

```diff
@@ -39,9 +39,14 @@ jobs:
           export PATH="$PATH:$PWD/exodus_core/dexdump/"
           python -m unittest discover -v -s exodus_core -p "test_*.py"
           python setup.py sdist
-      - name: Publish package to pypi on tag
-        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
-        uses: pypa/gh-action-pypi-publish@release/v1
-        with:
+  publish:
+     needs: build
```

Comment:
**question**: I have not used GitHub actions so much yet, does this mean that the `dist` folder from the `build` part will be available in the `publish` part?

Otherwise, the `publish` will fail

Reply:
With new jobs, other files are not keep.
We can use other solutions [action/cache](https://levelup.gitconnected.com/github-actions-how-to-share-data-between-jobs-fc1547defc3e) but i have any knowledge about that or use upload/download artifact between jobs (Give me path for files use by publish action).
Ok to keep one version (it's more easy and we keep only one job.